br-webui/index.js

```diff
@@ -24,6 +28,11 @@ app.use('/css', express.static(__dirname + '/node_modules/bootstrap-select/dist/
 app.use('/js', express.static(__dirname + '/node_modules/bootstrap-slider/dist'));
 app.use('/css', express.static(__dirname + '/node_modules/bootstrap-slider/dist/css'));
 
+// reverse proxy for mavlink
+app.all("/mavlink*", function(req, res) {
```

Comment:
It works with my tests here, but maybe `/mavlink` is not the best place, maybe `/rest/mavlink` ?
We have more pages in the future and one can be mavlink, it'll also be hard to identify what is a webpage and what is a rest api if we continue with this.

Reply:
well, what about /api/mavlink?

that will likely need some small changes to fix the path sent to mavlink2rest/

something like this should work: 
```
global.proxy.on('proxyReq', function(proxyReq, req, res, options){
   const rewritedPath = req.url.replace('/foo','/'); // can make any rule  
   proxyReq.path = rewritedPath;   
});
```

Reply:
Looks great, thanks! 
I'll let it approved and you can merge after the update.